### DISCUSSION TITLE
Q: 阿里云部署版本是没有更新了吗？

### DISCUSSION THREAD

**Discussion by XiaoPika** [walinejs/waline#1938] [Q&A]
### Describe the bug | 问题描述

上一次commit还是去年

### Website URL | 问题网站

github.com/orgs/walinejs

### Where your waline deploy? | 服务部署在哪里？

Aliyun FC

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.cn)

**Comment by lizheming** [comment]
模板不需要更新，重新部署就能最新了
